projects/documentation/content/_includes/partials/accessibility.njk

```diff
@@ -0,0 +1,100 @@
+ <div class="headerContainer">
+    <h2 id="accessibility" tabindex="-1" class="spectrum-Heading spectrum-Heading--sizeM">Accessibility testing status&nbsp;<sp-link class="header-anchor" href="#accessibility" aria-label="Accessibility" dir="ltr" focusable="">#</sp-link></h2>      
+    <sp-divider size="l" dir="ltr" role="separator"></sp-divider>
+</div>
+    <sp-table quiet>
+    <sp-table-head>
+        <sp-table-head-cell>Component</sp-table-head-cell>
+        <sp-table-head-cell>Accessibility test</sp-table-head-cell>
+        <sp-table-head-cell>Status</sp-table-head-cell>
+         <sp-table-head-cell>Source Code</sp-table-head-cell>
+    </sp-table-head>
+    <sp-table-body>
+        <sp-table-row value="row1">
+            <sp-table-cell><sp-link href="https://opensource.adobe.com/spectrum-web-components/components/{{componentName}}/#usage">{{componentName}}</sp-link></sp-table-cell>
+            <sp-table-cell>
+            <span id="defaulttest">Default state</span> 
+            <sp-overlay trigger="defaulttest@hover" placement="bottom">
```

Comment:
We're still working on https://github.com/adobe/spectrum-web-components/issues/3596 but, the accessibility here with a screen reader should be more like https://enhancement-docs-accessibility--spectrum-web-components.netlify.app/components/overlay/#example than it is now. I'm not sure if that's the lack of tab indexes on the triggers, or the duplicate IDs, or `type="auto"` when it should be `type="hint"`, or something else, but please take a closer look to ensure we've at least prepared this content to the best of the capabilities of the elements we're using.

Reply:
I checked and the accessibility is not there for badge component which in turn renders a label. So I have updated each element inside this table to render as a button to leverage the accessibility implementations of a button and tested it in screen reader. Please note this is a screen reader test. The UI has been updated accordingly.

<img width="1728" alt="Screenshot 2024-03-11 at 3 58 36 PM" src="https://github.com/adobe/spectrum-web-components/assets/8790510/68cd3997-2dc2-4590-a235-e0149f821479">